onnxruntime_extensions/tools/pre_post_processing/steps/vision.py

```diff
@@ -370,6 +381,12 @@ def _create_graph_for_step(self, graph: onnx.GraphProto, onnx_opset: int):
             # Split now requires the number of outputs to be specified even though that can be easily inferred...
             split_input_shape_attr += f", num_outputs = {len(dims)}"
             split_new_sizes_attr += ", num_outputs = 2"
+        
+        # Resize-18 has the attribute "not_larger/not_smaller" to specify the resize policy, however
+        # we want to support older opsets as well. 
+        ratio_resize_func = "ReduceMax"
+        if self.policy_ == "not_larger":
+            ratio_resize_func = "ReduceMin"
```

Comment:
should we also check the policy == 'not_smaller' so we can return an error if the user incorrectly tries to use an unsupported policy that the ONNX Resize op defines?